source/crud/read-operations/sort.txt

```diff
@@ -0,0 +1,82 @@
+============
+Sort Results
+============
+
+.. default-domain:: mongodb
+
+Use a sort to change the order in which operations return documents.
+Sorts tell MongoDB to order returned documents by the values of one or
```

Comment:
Would refer to sort in the singular form, especially since that is the name of the function that you are introducing. E.g. "Sort tells MongoDB...".